library/ssl_tls.c

```diff
@@ -4816,22 +4909,30 @@ static void ssl_bitmask_set( unsigned char *mask, size_t offset, size_t len )
     memset( mask + offset / 8, 0xFF, len / 8 );
 }
 
+#define BITMASK_CHECK_FAILED 0x75555555
```

Comment:
In this file, the value 0x75555555 is defined three times with different names and different meanings. 
```
#define PROPER_HS_FRAGMENT   0x75555555
#define BITMASK_CHECK_FAILED 0x75555555
#define PEER_CRT_CHANGED     0x75555555
```
I think this is not a very good approach. What do you think about replacing it with something like this:
```
typedef enum
{
    SCA_FALSE = 0x15555555,
    SCA_TRUE = 0x1aaaaaaa
} SCA_BOOLEAN;
```
Could `BITMASK_CHECK_FAILED` be replaced by some error code?

Reply:
All of these places are internal, static functions (if I'm not mistaken) that only were checked for a non-zero value. Adding error codes that won't be returned anywhere from the library is in my opinion unnecessary.